src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -175,7 +175,7 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                             }
                             else
                             {
-                                _logger.LogWarning(0, ex, $"Unable to bind to {parsedAddress.ToString()} on the IPv6 loopback interface.");
```

Comment:
Yeah I was thinking about that too. We know it's a benign error specifically on Azure, but it's still a potentially legit error in other environments.